addons/web_editor/static/src/scss/wysiwyg_snippets.scss

```diff
@@ -2838,10 +2861,10 @@ we-select.o_grid we-toggler {
 
         .o_snippet_preview_wrap {
             min-height: 100px;
-            transition: all 150ms cubic-bezier(0.65, 0.05, 0.36, 1);
-            box-shadow: 0 0.5rem 6rem rgba(0, 0, 0, 0.2);
-            margin-bottom: 96px !important;
+            box-shadow: 0 0 6rem rgba(0, 0, 0, 0.16);
+            margin: map-get($spacers, 5) (map-get($spacers, 5) * .5) (map-get($spacers, 3) * 4);
             background-color: var(--body-bg);
+            transform: scale(.98);
```

Comment:
why we need this scale ?

Reply:
So, since we greatly reduce the margin between the elements in order to solve the "container padding" issue, we still need a way to distantiate them within the modal. Scale does the trick 😉
`.98` it's quite arbitrary, but it works nice till 4K 👍

|`m-5` | smaller horizontal margins + `scale(.98)` |
|--------|--------|
| ![image](https://github.com/user-attachments/assets/05e971e8-840e-4c28-9cbf-a2041547edef) | ![image](https://github.com/user-attachments/assets/74f5db8a-e82a-47e4-973f-d9f7d6f22557) |